spec/coordinate_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'pry'
+require_relative '../lib/coordinate.rb'
+
+describe Coordinate do
+  let(:x) { 0 }
+  let(:y) { 0 }
+
+  it 'has starting coordinates' do
+    expect(coordinate.x).to eq 0
+    expect(coordinate.y).to eq 0
+  end
+
+  it 'adds 1 to y' do
+    expect(coordinate.up.x).to eq 0
+    expect(coordinate.up.y).to eq 1
+  end
+
+  it 'removes 1 to y' do
+    expect(coordinate.down.x).to eq 0
+    expect(coordinate.down.y).to eq -1
+  end
+
+  it 'adds 1 to x' do
+    expect(coordinate.right.x).to eq 1
+    expect(coordinate.right.y).to eq 0
```

Comment:
Notice the duplication in asserting on the x-coordinate and y-coordinate. How might we minimise that duplication?